frontend/assets/icons/viewer/align.svg.tsx

```diff
@@ -0,0 +1,28 @@
+/**
```

Comment:
Some tool tip changes

![image](https://github.com/3drepo/3drepo.io/assets/11945337/a77ffec6-d986-4a7d-873e-00ad395ed9fe)

Can we call this `Move`?

![image](https://github.com/3drepo/3drepo.io/assets/11945337/56106fd6-ecd3-4028-acba-775596af754c)

This should be `Remove` or `Delete`.

![image](https://github.com/3drepo/3drepo.io/assets/11945337/731726da-5eea-4fad-a74c-6b537ca98bea)
`Align to surface`

![image](https://github.com/3drepo/3drepo.io/assets/11945337/fff64743-7b03-4563-b127-bd37dc9afd6d)

`Resize`

![24-01-04 02-36-27](https://github.com/3drepo/3drepo.io/assets/11945337/6cd2196e-1c98-4488-868a-d3de6ec20456)
Can we call this `Sectioning`, `Section by plane`, `Section by Box`

Reply:
Agreed the disable look isn't prominent enough. we should look into making it more obvious

Reply:
> Agreed the disable look isn't prominent enough. we should look into making it more obvious

@The-Daniel don't think there was any changes for this?